### PR TITLE
SQL initial update

### PR DESCRIPTION
**Pull request by Trevor27**
Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## I solemnly swear that:

- [x] I ran the hugo server and looked at my changed in the browser with my own eyes
- [x] I ran the linter and there were no errors
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by Sbonelo01 — CHANGES_REQUESTED** [review]
Looks good to me, but

For many learners, SQL is relatively a new thing. Can you please point learners to a good SQL best practices resource.

Also, a resources about Docker and  docker-compose. Many learners configure docker-compose.yaml file to make use of the adminer image but still instruct reviewers to download and install Postgres on their local machines in the documentation. It seems they aren't sure about the relationship of these technologies

File: content/sql/shop-project/_index.md [hunk 1]
```diff
@@ -165,8 +166,16 @@ NB! Be sure to label you answers by putting the question above the code that ans
...
+  
+  **NB! Many learners configure the docker-compose.yaml file to make use of the Adminer image but still instruct reviewers to download and install Postgres on their local machines in the documentation. This indicates a misunderstanding of the relationship between these technologies.**
+
+  To Clarify:
+  - Docker allows you to run applications in isolated containers.
+  - Docker Compose is a tool for defining and running multi-container Docker applications.
+  - Using Docker and Docker Compose, you can set up your entire environment, including Postgres and Adminer, without requiring any local installations on the reviewer’s machine.
+  
+  Ensure the documentation reflects this setup correctly, so reviewers do not need to install Postgres locally. Refer to the following resource for more information: [How to Set Up Postgres and Adminer with Docker Compose](https://codepruner.com/how-to-run-postgresql-and-adminer-or-pgadmin-with-docker-compose/)
```
Comment (Ngoakor12) [inline content/sql/shop-project/_index.md]: This part seems is important and should probably be higher up in the content, maybe under the `Installation` information.

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
Mostly looks good to me

File: content/sql/shop-project/_index.md (hunk 1 above)
Comment (Ngoakor12) [inline content/sql/shop-project/_index.md]: We also have a resource for setting docker up that's in the prerequisites so maybe relink that too(http://localhost:1313/docker/intro-to-docker/)

**Review by Ngoakor12 — APPROVED** [review]

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**